### ISSUE TITLE
Can I ingest a JSON data to the API url using this library?

### ISSUE THREAD

**Issue by debuggerrr** [sourav-mazumder/Data-Science-Extensions#15]
https://stackoverflow.com/questions/72565095/how-to-pass-columns-as-a-json-record-to-the-api-method-using-pyspark

this problem will give you an overview of what I am trying to do. So, is it possible to achieve my goal using this library?

**Comment by sourav-mazumder** [comment]
Yes u can ideally. But this library is not maintained for a long time.
